docs/source/en/api/pipelines/perturbed_attention_guidance.md

```diff
@@ -0,0 +1,61 @@
+<!--Copyright 2024 The HuggingFace Team. All rights reserved.
```

Comment:
We don't add the community pipelines here. Just including them in the README is enough :)

Reply:
Thank you so much for the feedback! I have excluded the docs from the pull request.